src/morse_code.rb

```diff
@@ -0,0 +1,44 @@
+# Create a method to decode a Morse code character
+
+def decode_char(char)
+  morse_chars_codes = {
+    '.-' => 'A', '-...' => 'B', '-.-.' => 'C', '-..' => 'D', '.' => 'E',
+    '..-.' => 'F', '--.' => 'G', '....' => 'H', '..' => 'I', '.---' => 'J',
+    '-.-' => 'K', '.-..' => 'L', '--' => 'M', '-.' => 'N', '---' => 'O',
+    '.--.' => 'P', '--.-' => 'Q', '.-.' => 'R', '...' => 'S', '-' => 'T',
+    '..-' => 'U', '...-' => 'V', '.--' => 'W', '-..-' => 'X', '-.--' => 'Y',
+    '--..' => 'Z'
+  }
+  morse_chars_codes.each do |key, value|
+    return value if key == char
+  end
+end
+
+puts decode_char('.-')
+
+# Create a method to decode an entire word in Morse code
+
+def decode_word(word)
+  myword = []
+  char = word.split
+  char.each do |c|
+    myword << decode_char(c)
+  end
+  myword = myword.join
+end
+
+puts decode_word('-- -.--')
+
+# # Create a method to decode the entire message in Morse code
+
+def decode(message)
+  mymessage = []
+  word = message.split('   ')
+  word.each do |w|
+    mymessage << decode_word(w)
+  end
+  puts mymessage = mymessage.join(' ')

```

Comment:
- [ ] due to the `puts` statement in line 40 the return values are not correct as you can see in the below screenshot:
![Screenshot 2022-04-01 225345](https://user-images.githubusercontent.com/39240820/161340293-7187e2e1-552b-4452-8ed0-1d49bf1bf2c4.png)
according to the assignment, the methods should return the string values instead of printing them. therefore please remove the `puts` statement in this line